ddapm_test_agent/agent.py

```diff
@@ -509,14 +576,19 @@ def make_app(
             web.put("/v0.5/traces", agent.handle_v05_traces),
             web.post("/v0.6/stats", agent.handle_v06_tracestats),
             web.put("/v0.6/stats", agent.handle_v06_tracestats),
+            web.post(
```

Comment:
`put` is probably also supported but i'm fine to leave this for a follow up

Reply:
The API schema/docs only say `POST`, so I'd rather leave that as-is.